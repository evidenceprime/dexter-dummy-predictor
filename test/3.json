{
  "text": "Blood pressure and pulse\n Although decreases in systolic blood pressure occurred with either liraglutide 1.2 or 1.8 mg (2.6–2.8 mmHg), they were not significantly different from placebo or rosiglitazone (0.9–2.3 mmHg). Reductions in diastolic blood pressure also occurred with all treatments (0.7–1.4 mmHg), with no significant differences between treatments. Pulse increases above baseline ranged from 2 to 4 beats/min with the three doses of liraglutide and 1 beat/min with rosiglitazone, while pulse decreased by 1 beat/min with placebo. Changes in pulse for all doses of liraglutide were significant vs. placebo (P ≤ 0.002). This also was true with either liraglutide 1.8 or 1.2 mg compared with rosiglitazone (P < 0.01).",
  "sections": [
  ]
}
