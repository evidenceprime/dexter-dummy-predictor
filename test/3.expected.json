{
  "mentions": [
    {
      "id": 1,
      "tag": "CARDINAL",
      "offsets": [
        105
      ],
      "words": [
        "1.2"
      ]
    },
    {
      "id": 3,
      "tag": "CARDINAL",
      "offsets": [
        112
      ],
      "words": [
        "1.8"
      ]
    },
    {
      "id": 5,
      "tag": "CARDINAL",
      "offsets": [
        120
      ],
      "words": [
        "2.6–2.8"
      ]
    },
    {
      "id": 7,
      "tag": "CARDINAL",
      "offsets": [
        293
      ],
      "words": [
        "0.7–1.4"
      ]
    },
    {
      "id": 9,
      "tag": "CARDINAL",
      "offsets": [
        403
      ],
      "words": [
        "2"
      ]
    },
    {
      "id": 11,
      "tag": "CARDINAL",
      "offsets": [
        408
      ],
      "words": [
        "4"
      ]
    },
    {
      "id": 13,
      "tag": "CARDINAL",
      "offsets": [
        429
      ],
      "words": [
        "three"
      ]
    },
    {
      "id": 15,
      "tag": "CARDINAL",
      "offsets": [
        460
      ],
      "words": [
        "1"
      ]
    },
    {
      "id": 17,
      "tag": "CARDINAL",
      "offsets": [
        516
      ],
      "words": [
        "1"
      ]
    },
    {
      "id": 19,
      "tag": "PERSON",
      "offsets": [
        608
      ],
      "words": [
        "placebo"
      ]
    },
    {
      "id": 21,
      "tag": "MONEY",
      "offsets": [
        621
      ],
      "words": [
        "0.002"
      ]
    },
    {
      "id": 23,
      "tag": "CARDINAL",
      "offsets": [
        672
      ],
      "words": [
        "1.8"
      ]
    },
    {
      "id": 25,
      "tag": "CARDINAL",
      "offsets": [
        679
      ],
      "words": [
        "1.2"
      ]
    }
  ],
  "equivalences": [
    {
      "id": 0,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            7
          ],
          "text": "0.7–1.4"
        }
      ]
    },
    {
      "id": 1,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            15,
            17
          ],
          "text": "1"
        }
      ]
    },
    {
      "id": 2,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            1,
            25
          ],
          "text": "1.2"
        }
      ]
    },
    {
      "id": 3,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            3,
            23
          ],
          "text": "1.8"
        }
      ]
    },
    {
      "id": 4,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            9
          ],
          "text": "2"
        }
      ]
    },
    {
      "id": 5,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            5
          ],
          "text": "2.6–2.8"
        }
      ]
    },
    {
      "id": 6,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            11
          ],
          "text": "4"
        }
      ]
    },
    {
      "id": 7,
      "tag": "CARDINAL",
      "items": [
        {
          "ids": [
            13
          ],
          "text": "three"
        }
      ]
    },
    {
      "id": 8,
      "tag": "MONEY",
      "items": [
        {
          "ids": [
            21
          ],
          "text": "0.002"
        }
      ]
    },
    {
      "id": 9,
      "tag": "PERSON",
      "items": [
        {
          "ids": [
            19
          ],
          "text": "placebo"
        }
      ]
    }
  ]
}